gnirehtet.cmd

```diff
@@ -0,0 +1,150 @@
+@ECHO OFF
+CALL :locate_file ADB %~dp0adb.exe "%LOCALAPPDATA%\Android\sdk\platform-tools\adb.exe"
```

Comment:
`:locate_file` is now first searching in `%PATH%` before the input directories list.
I don't like touching environment variables within my code. I can do with `SETLOCAL` and `ENDLOCAL` but then I should write a new subroutine (function) for this. So I preferred just changing  `:locate_file` to use `where`. BTW, the script is only working on Windows 2003 and up.